src/dlgProfilePreferences.cpp

```diff
@@ -1958,3 +2206,34 @@ void dlgProfilePreferences::slot_resetThemeUpdateLabel()
     theme_download_label->hide();
     theme_download_label->setText(tr("Updating themes from colorsublime.com..."));
 }
+
+void dlgProfilePreferences::slot_handleHostAddition(Host* pHost, const quint8 count)
```

Comment:
Mind adding a comment here explaining what the function's purpose is? "handle host addition"... to do what?